docs/source/_notebooks/xgboost-titanic.rst

```diff
@@ -0,0 +1,2608 @@
+                        <td style="padding: 0 1em 0 0.5em; text-align: right; border: none;">
+                            0.0151
+                            
+                        </td>
+                        <td style="padding: 0 0.5em 0 0.5em; text-align: left; border: none;">
+                            Name__<span style="background-color: hsl(120, 80%, 70%); margin: 0 0.1em 0 0.1em" title="A space symbol">&emsp;</span>Mrs
+                        </td>
+                    </tr>
+                
+                    <tr style="background-color: hsl(120, 100.00%, 96.60%); border: none;">
+                        <td style="padding: 0 1em 0 0.5em; text-align: right; border: none;">
+                            0.0151
+                            
+                        </td>
+                        <td style="padding: 0 0.5em 0 0.5em; text-align: left; border: none;">
+                            All__Pclass=1
+                        </td>
+                    </tr>
+                
+                    <tr style="background-color: hsl(120, 100.00%, 96.60%); border: none;">
+                        <td style="padding: 0 1em 0 0.5em; text-align: right; border: none;">
+                            0.0151
+                            
+                        </td>
+                        <td style="padding: 0 0.5em 0 0.5em; text-align: left; border: none;">
+                            All__Embarked=S
+                        </td>
+                    </tr>
+                
+                    <tr style="background-color: hsl(120, 100.00%, 96.87%); border: none;">
+                        <td style="padding: 0 1em 0 0.5em; text-align: right; border: none;">
+                            0.0134
+                            
+                        </td>
+                        <td style="padding: 0 0.5em 0 0.5em; text-align: left; border: none;">
+                            All__Pclass=2
+                        </td>
+                    </tr>
+                
+                
+                    
+                        <tr style="background-color: hsl(120, 100.00%, 96.87%); border: none;">
+                            <td colspan="2" style="padding: 0 0.5em 0 0.5em; text-align: center; border: none;">
+                                <i>&hellip; 34 more &hellip;</i>
+                            </td>
+                        </tr>
+                    
+                
+                </tbody>
+            </table>
+        
+    
+        
+    
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+    
+    
+
+
+
+
+We see that now there are a lot of features that come from the *Name*
+field (in fact, a classifier based on *Name* alone gives about 0.79
+accuracy). Name features listed in this way are not very informative,
+they make more sense when we check out predictions. We hide missing
+features here because there are a lot of missing features in text, but
```

Comment:
wow, I was pretty sure that is wrong, but it's actually correct, thanks for the catch!